runtime/kusama/src/lib.rs

```diff
@@ -814,6 +815,9 @@ impl InstanceFilter<Call> for ProxyType {
 				Call::Staking(..) | Call::Utility(utility::Call::batch(..))
 					| Call::Utility(utility::Call::as_limited_sub(..))
 			),
+			ProxyType::RegistrarJudgement => matches!(c,
```

Comment:
probably worth including `utility.batch` also.

Reply:
yes, that's exactly what I want to add :) I was looking for how to add batch but only a given method.

I have also (not pushed yet) the change for westend and polkadot.

Reply:
@chevdor if you directly add `Call::Utility(utility::Call::batch(..))` as is done above, the other filters will continue to apply through to the batch call.

Reply:
i.e. you will only be able to batch to make the specific calls you have whitelisted. You of course can verify this with a dev node or something.